Project.toml

```diff
@@ -36,6 +36,7 @@ StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 StrideArrays = "d1fa6d79-ef01-42a6-86c9-f7c551f8593b"
 StructArrays = "09ab397b-f2b6-538f-b94a-2f83cf4a842a"
 SummationByPartsOperators = "9f78cca6-572e-554e-b819-917d2f1cf240"
+T8code = "d0cc0030-9a40-4274-8435-baadcfd54fa1"
```

Comment:
This also needs a compat bound. Where is T8code.jl hosted - it's not in the Trixi.jl framework, is it?

Reply:
https://github.com/DLR-AMR/T8code.jl

Reply:
Added a compat entry for T8code.jl.